tests/test_metadata.py

```diff
@@ -429,7 +428,9 @@ def test_disturl(
             )
         )
     elif OS_VERSION == "16.0":
-        if "opensuse.org" in container.container.get_base().url:
+        if container.container.get_base().url.startswith(
+            "registry.opensuse.org"
```

Comment:
There's no protocol in the URL?

Reply:
it's not really a url (dunno why pytest-container calls it that way). it's basically what you would otherwise specify in the `FROM ` statement.